plugins/ccxtExchange_test.go

```diff
@@ -556,24 +557,17 @@ func TestCancelOrder_Ccxt(t *testing.T) {
 }
 
 func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
-	// coinbasepro gives incorrect precision values so we do not test it here
```

Comment:
Why modify this file?
(Wondering how it’s related to the change)

Reply:
just to get the build green

Reply:
Once we merge this fix we can rebase and this will work.